Orange/widgets/visualize/owscoringsheetviewer.py

```diff
@@ -510,15 +515,32 @@ def _extract_data_from_model(self, classifier):
         all_risks = 1 / (1 + np.exp(-all_scaled_scores))
 
         self.attributes = attributes
+        self._base_coefficients = coefficients
+        self._base_all_scores = all_scores.tolist()
+        self._base_all_risks = (all_risks * 100).tolist()
         self.coefficients = coefficients
```

Comment:
Are these three lines still necessary? These attributes are set in `self._adjust_for_target_class`, which is called a few lines later.